modules/virt-overriding-default-fs-overhead-value.adoc

```diff
@@ -6,43 +6,42 @@
 [id="virt-overriding-default-fs-overhead-value_{context}"]
 = Overriding the default file system overhead value
 
-Change the amount of persistent volume claim (PVC) space that the Containerized Data Importer (CDI) reserves for file system overhead by editing the `spec.config.filesystemOverhead` attribute of the `CDI` object.
+Change the amount of persistent volume claim (PVC) space that OpenShift Virtualization reserves for file system overhead by editing the `spec.filesystemOverhead` attribute of the `HCO` object.
 
 .Prerequisites
 
 * Install the OpenShift CLI (`oc`).
 
 .Procedure
 
-. Open the `CDI` object for editing by running the following command:
+. Open the `HCO` object for editing by running the following command:
 +
 [source,terminal]
 ----
-$ oc edit cdi
+$ oc edit hco
 ----
 
-. Edit the `spec.config.filesystemOverhead` fields, populating them with your chosen values:
+. Edit the `spec.filesystemOverhead` fields. You can choose from either or both a global value that will apply to all storage classes, or a storage class specific value.
 +
 [source,yaml]
 ----
 ...
 spec:
-  config:
-    filesystemOverhead:
-      global: "<new_global_value>" <1>
-      storageClass:
-        <storage_class_name>: "<new_value_for_this_storage_class>" <2>
+  filesystemOverhead:
+    global: "<new_global_value>" <1>
+    storageClass:
+      <storage_class_name>: "<new_value_for_this_storage_class>" <2>
 ----
-<1> The file system overhead percentage that CDI uses across the cluster. For example, `global: "0.07"` reserves 7% of the PVC for file system overhead.
-<2> The file system overhead percentage for the specified storage class. For example, `mystorageclass: "0.04"` changes the default overhead value for PVCs in the `mystorageclass` storage class to 4%.
+<1> The file system overhead percentage that OpenShift Virtualization uses across the cluster. For example, `global: "0.07"` reserves 7% of the PVC for file system overhead.
+<2> The file system overhead percentage for the specified storage class. For example, `mystorageclass: "0.04"` changes the default overhead value for PVCs in the `mystorageclass` storage class to 4%. If both a global value and a storage class value appear, the storage class specific value will be used.
```

Comment:
If both a global value and a storage class value appear, the storage class specific value will be used. --> 
Change to:
If both a global value and a storage class value are configured, the storage class specific value will be used.